<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Petr Koup√Ω</author>
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="1" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBoxAnimation">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Animation</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelAnimationLength">
       <property name="text">
        <string>Length:</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxAnimationLength</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBoxAnimationLength">
       <property name="suffix">
        <string> msec</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>30000</number>
       </property>
       <property name="value">
        <number>700</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelAnimationStyle">
       <property name="text">
        <string>Style:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxAnimationStyle</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBoxAnimationStyle"/>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBoxLayout">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Layout</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelLayoutingMode">
       <property name="text">
        <string>Layouting mode:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxLayoutingMode</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboBoxLayoutingMode"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelLayouter">
       <property name="text">
        <string>Layouter:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxLayouter</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="comboBoxLayouter"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelNodeDisplacement">
       <property name="text">
        <string>Node displacement:</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxNodeDisplacement</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="spinBoxNodeDisplacement">
       <property name="minimum">
        <number>20</number>
       </property>
       <property name="maximum">
        <number>90</number>
       </property>
       <property name="value">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelGridOffset">
       <property name="text">
        <string>Grid offset:</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxGridOffset</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QSpinBox" name="spinBoxGridOffset">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>14</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBoxValidation">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Validation</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="labelValidator">
       <property name="text">
        <string>Validator:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxValidator</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QComboBox" name="comboBoxValidator"/>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBoxVisual">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Visual</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="labelNodeDescription">
       <property name="text">
        <string>Node description:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxNodeDescription</cstring>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QComboBox" name="comboBoxNodeDescription"/>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="labelControlsVisibility">
       <property name="text">
        <string>Controls visibility:</string>
       </property>
       <property name="buddy">
        <cstring>checkBoxControlsVisibility</cstring>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QCheckBox" name="checkBoxControlsVisibility">
       <property name="text">
        <string>visible</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="labelInteractiveCapturing">
       <property name="text">
        <string>Interactive capturing:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QCheckBox" name="checkBoxInteractiveCapturing">
       <property name="text">
        <string>interactive</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonDefault">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save as Default</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBoxAnimationLength</tabstop>
  <tabstop>comboBoxAnimationStyle</tabstop>
  <tabstop>comboBoxLayoutingMode</tabstop>
  <tabstop>comboBoxLayouter</tabstop>
  <tabstop>spinBoxNodeDisplacement</tabstop>
  <tabstop>spinBoxGridOffset</tabstop>
  <tabstop>comboBoxValidator</tabstop>
  <tabstop>comboBoxNodeDescription</tabstop>
  <tabstop>checkBoxControlsVisibility</tabstop>
  <tabstop>buttonOK</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
